components/collection/drop/modal/shared/SuccessfulDrop.vue

```diff
@@ -125,21 +23,23 @@ const props = defineProps<{
 
 const { $i18n } = useNuxtApp()
 const { toast } = useToast()
-const { getExtrinsicUrl } = useExplorer()
-const { shareOnX, shareOnTelegram } = useSocialShare()
 
 const sharingTxt = $i18n.t('sharing.nft')
 
-const txUrl = computed(() =>
-  getExtrinsicUrl(
-    props.mintedNft.txHash || '',
-    props.mintedNft.chain as Prefix,
-  ),
-)
+const txHash = computed(() => props.mintedNft.txHash || '')
+const share = computed(() => ({ text: sharingTxt, url: nftFullUrl.value }))
 
-const collectionUrl = computed(
-  () => `/${props.mintedNft.chain}/collection/${props.mintedNft.collection}`,
-)
+const actionButtons = computed(() => ({
```

Comment:
Could you check if it still works without `computed`?

Reply:
checked to be sure , it doens't , `cantList` is not updated if `actionButtons ` it's just an object. 

![CleanShot 2024-02-05 at 09 56 36@2x](https://github.com/kodadot/nft-gallery/assets/44554284/7dd501ff-4df2-4062-8aae-ace4e1088fd8)